dlt/common/schema/utils.py

```diff
@@ -440,7 +441,7 @@ def merge_columns(
     * incomplete columns in `columns_a` that got completed in `columns_b` are removed to preserve order
     """
     if columns_partial is False:
-        raise NotImplementedError("columns_partial must be False for merge_columns")
+        raise NotImplementedError("`columns_partial` must be `False` for `merge_columns`")
```

Comment:
issue: I don't think the error message is helpful here, as columns_partial being false is exactly what raises the error here, isn't it

Reply:
not a problem this pr created , but it could fix it